adapters/ix/ix_test.go

```diff
@@ -276,3 +274,63 @@ func TestMakeRequestsErrIxDiag(t *testing.T) {
 	_, errs := bidder.MakeRequests(req, nil)
 	assert.Len(t, errs, 1)
 }
+
+func TestMoveSid(t *testing.T) {
```

Comment:
Requesting you to please move these test case to use json framework as this is the recommended way to test a bid adapter code.